cfme/tests/services/test_rest_services.py

```diff
@@ -778,3 +778,77 @@ def test_edit_orchestration_templates(self, rest_api, orchestration_templates, f
             assert edited[i].description == new[i]['description']
             orchestration_templates[i].reload()
             assert orchestration_templates[i].description == new[i]['description']
+
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.8')
```

Comment:
We should have a better way to do this shortly, but I believe for now using version.current_version in the pytest decorators is acceptable.